pkg/platform/runtime/artifactcache/testhelper.go

```diff
@@ -0,0 +1,53 @@
+package artifactcache
+
+// This file exists solely to export private data from ArtifactCache in order to run integration
+// tests in an outside package.
+
+import (
+	"github.com/ActiveState/cli/pkg/platform/runtime/artifact"
+)
+
+type testArtifactCache struct {
+	cache *ArtifactCache
+}
+
+// TestNew is only meant to be called from tests. Use New() instead.
+func TestNew(dir string, maxSize int64) (*testArtifactCache, error) {
```

Comment:
Please consider following convention here, which would lead to this being named NewTestArtifactCache.